example/data.js

```diff
@@ -73,6 +73,22 @@ const bumpSyncToken = function(cal) {
   cal.syncToken = parts.slice(0, -1).join('/') + '/' + (parseInt(parts[parts.length - 1]) + 1);
 };
 
+const eventFilter = (start, end, eventStart, eventEnd) =>
+  (!end || end < eventStart) || (!start || start > eventEnd)
+
+const recurringEventFilter = (start, end, event) => {
```

Comment:
If possible, please add types to function parameters.

Reply:
This is a `js`file, I therefore omitted the typing. Just like the other functions do in this file, too.
I'm guess it would make sense to rewrite this file and use `ts-node` to execute it. I guess a change for another PR

Reply:
Ahh my mistake.